src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -560,8 +563,8 @@ private _SqlMetaDataSet(_SqlMetaDataSet original)
         {
             this.id = original.id;
             // although indexMap is not immutable, in practice it is initialized once and then passed around
```

Comment:
This comment is now out-of-date.

Reply:
removed, and removed the unneeded this prefixes around it too.